client_cli/pytests/test/accounts/test_accounts_query_filters.py

```diff
@@ -1,51 +1,66 @@
 import json
-import allure
 
-from src.client_cli import iroha, client_cli
+import allure  # type: ignore
+
+from src.client_cli import client_cli, iroha
+
 
 # using existing account to have at least one account in response
-def test_filter_by_domain(GIVEN_new_one_existing_account):
+def test_filter_by_domain(GIVEN_registered_account):
     def condition():
-        domain = GIVEN_new_one_existing_account.domain
-        with allure.step(
-                f'WHEN client_cli query accounts '
-                f'in the "{domain}" domain'):
-            accounts = iroha.list_filter(f'{{"Identifiable": {{"EndsWith": "@{domain}"}}}}').accounts()
+        domain = GIVEN_registered_account.domain
         with allure.step(
-                f'THEN Iroha should return only accounts with this domain'):
+            f"WHEN client_cli query accounts " f'in the "{domain}" domain'
+        ):
+            accounts = iroha.list_filter(
+                f'{{"Identifiable": {{"EndsWith": "@{domain}"}}}}'
+            ).accounts()
```

Comment:
I think it would be more readable if `list_filter` accepts native Python `dict` rather than a raw JSON string:

```suggestion
            accounts = iroha.list_filter(
                { "Identifiable": { "EndsWith": f"@{domain}" } }
            ).accounts()
```

Reply:
done